packages/autoconf.rb

```diff
@@ -3,7 +3,7 @@
 class Autoconf < Package
   description 'Autoconf is an extensible package of M4 macros that produce shell scripts to automatically configure software source code packages.'
   homepage 'http://www.gnu.org/software/autoconf/'
-  version '2.69'
+  version '2.69' # Do not change, required for latest glibc version on 5-29-18
   source_url 'ftp://ftp.gnu.org/gnu/autoconf/autoconf-2.69.tar.xz'
```

Comment:
Can we change this to `source_url 'https://ftpmirror.gnu.org/autoconf/autoconf-2.69.tar.xz'`?

Reply:
Sure.